.gitignore

```diff
@@ -0,0 +1,3 @@
+build
+*.txt.user
+*thrift.cmake
```

Comment:
I never used git ignore as (ideally) one does not add to the tracked files the ones above. but this is useful anyway.
Thanks